pkgs/development/python-modules/sphinxcontrib-autoapi/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, astroid
+, jinja2
+, sphinx
+, pyyaml
+, unidecode
+, mock
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autoapi";
+  version = "1.5.0";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1hq96s0kg5v6zkb813k4w11rx0n1hv9iqyllsj24sa0qfsgbdzwm";
+  };
+
+  propagatedBuildInputs = [ astroid jinja2 pyyaml sphinx unidecode ];
+
+  checkInputs = [
+    mock
+    pytest
+  ];
+
+  checkPhase = ''
+    pytest
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/readthedocs/sphinx-autoapi";
+    description = "Sphinx AutoAPI provides 'autodoc' style documentation for multiple programming languages without needing to load, run, or import the project being documented.";
```

Description should be short and succinct
```suggestion
    description = "Provides 'autodoc' style documentation";
```

if you want to have a long description, you can use longDescription

---

https://nixos.org/manual/nixpkgs/stable/#sec-standard-meta-attributes

---

Done. I didn't know about longDescription, thx!